views/js/payment-components/one-click.js

```diff
@@ -80,32 +88,35 @@ jQuery(function ($) {
                     'language': language,
                     'javaEnabled': javaEnabled
                 }
-            }, oneClickPaymentMethod, paymentForm);
+            }, storedPaymentMethod, paymentForm);
         });
     });
 
     /**
      * Renders checkout card component
      */
-    function renderOneClickComponent(oneClickPaymentMethod) {
-        var card = window.adyenCheckout.create('card', {
-            type: oneClickPaymentMethod.type,
-            oneClick: true,
-            details: oneClickPaymentMethod.details,
-            storedDetails: oneClickPaymentMethod.storedDetails,
-
+    function renderStoredPaymentComponent(storedPaymentMethod) {
+        var card = window.adyenCheckout.create('card', Object.assign(
```

Comment:
`Object.assign` is not available in any version of Internet Explorer

Reply:
indeed, I will have to find a solution for this one